typescript/infra/config/environments/mainnet3/warp/warpIds.ts

```diff
@@ -5,6 +5,7 @@ export enum WarpRouteIds {
   ArbitrumEthereumZircuitAMPHRETH = 'AMPHRETH/arbitrum-ethereum-zircuit',
   ArbitrumNeutronEclip = 'ECLIP/arbitrum-neutron',
   ArbitrumNeutronTIA = 'TIA/arbitrum-neutron',
+  ArbitrumTreasureMAGIC = 'MAGIC/arbitrum-treasure',
```

Comment:
Do we need to update .registryrc?

Reply:
isn't this already in main without a registry update https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/4998